misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/BinaryExpressionEvaluator.java

```diff
@@ -104,6 +114,12 @@ private BExpressionValue performOperation(BExpressionValue lhs, BExpressionValue
                 return logicalOR(lVar, rVar);
             case ELVIS_TOKEN:
                 return conditionalReturn(lVar, rVar);
+            case DOUBLE_EQUAL_TOKEN:
+            case NOT_EQUAL_TOKEN:
+                return checkValueEquality(lVar, rVar, operatorType);
+            case TRIPPLE_EQUAL_TOKEN:
```

Comment:
What is the relevant action if the operatorType is `DOUBLE_EQUAL_TOKEN` and TRIPPLE_EQUAL_TOKEN?
Are we planning to implement these in future?

Reply:
@praveennadarajah You might have missed that both of `DOUBLE_EQUAL_TOKEN` and `NOT_EQUAL_TOKEN` cases fall into `checkValueEquality` method.

Reply:
Okay, Noted.